python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -121,6 +120,32 @@ def from_pretrained(cls,
         optimize_model = kwargs.pop("optimize_model", True)
         user_quantization_config = kwargs.pop("quantization_config", None)
 
+        if user_quantization_config is not None and \
+                "BitsAndBytesConfig" in str(user_quantization_config.__class__):
```

Comment:
should we use `isinstance` instead?

Reply:
If using `isinstance`, we need to install BitsAndBytesConfig libs on bigdl-llm. Should we do that?